runtime/packages/core-modules/assembly/helpers.ts

```diff
@@ -0,0 +1,67 @@
+import { Hash, CompactInt, ScaleString, Bytes } from 'as-scale-codec';
+
+export class Helpers{
+    static getDefaultBlockHash(): Map<CompactInt, Hash>{
+        const defaultMap: Map<CompactInt, Hash> = new Map();
+        defaultMap.set(new CompactInt(0), Helpers.getPopulatedHash(69));
+        return defaultMap;
+    }
+
+    /**
+     * Converts storage key to u8a
+     * @param modPrefix module prefix
+     * @param stgPrefix storage prefix
+     */
+    static stringsToU8a(args: string[]): u8[]{
+        let result: u8[] = [];
+        for (let i=0; i<args.length ; i++){
+            const strScale = new ScaleString(args[i]); 
+            result = result.concat(strScale.toU8a());
+        }
+        return result;
+    }
+    /**
+     * 
+     * @param byte default byte
+     */
+    static getPopulatedHash(byte: u8): Hash {
+        const hashBytes: u8[] = new Array<u8>(32);
+        hashBytes.fill(byte);
+        return Hash.fromU8a(hashBytes);
+    }
+    /**
+     * Converts SCALE encoded bytes to Map<CompactInt, Hash> 
+     * @param input SCALE encoded Map<CompactInt, Hash>
+     */
+    static blockHashFromU8Array(input: u8[]): Map<CompactInt, Hash>{
+        const blockHash: Map<CompactInt, Hash> = new Map<CompactInt, Hash>();
```

Comment:
Instead of returning Map with CompactInt and Hash, you could create a new model and instantiate it. It would be more clean